### PR TITLE
feat: remove license api

### PR DESCRIPTION
**Pull request by CeerDecy**
#### What this PR does / why we need it:

forward “remove license” api

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/issues/all?id=577752&iterationID=12783&type=TASK)

#### Specified Reviewers:

/assign @iutx 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |      forward “remove license” api        |
| 🇨🇳 中文    |      转发license-agent的“移除License”接口        |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: internal/core/openapi/legacy/api/apis/license_agent/agent_remove_license_post.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+import (
+	"net/http"
+
+	"github.com/erda-project/erda/apistructs"
+	"github.com/erda-project/erda/internal/core/openapi/legacy/api/apis"
+)
+
+var AGENT_REMOVE_LICENSE_POST = apis.ApiSpec{
+	Path:         "/api/licenses/actions/remove",
```
Comment (iutx) [inline internal/core/openapi/legacy/api/apis/license_agent/agent_remove_license_post.go:25]: /api/licenses/actions 
DELETE method

**Review by iutx — CHANGES_REQUESTED** [review]
@CeerDecy

File: apistructs/license_agent.go [hunk 2]
```diff
@@ -8,3 +22,11 @@ type RegisterLicenseRequest struct {
 type RegisterLicenseResponse struct {
 	Id int64 `json:"id"`
 }
+
+type RemoveLicenseRequest struct {
```
Comment (iutx) [inline apistructs/license_agent.go:26]: Keyword `Remove` rename to `Delete` better.

**Review by iutx — CHANGES_REQUESTED** [review]
@CeerDecy

**Review by iutx — APPROVED** [review]
@sfwn PTAL.

**Review by sfwn — APPROVED** [review]

**Pull request merged**